src/sdk/signature.ts

```diff
@@ -1,20 +1,25 @@
 export async function verifySignature(publicKeyPem: string, payload: unknown, signature: string) {
-  const pemContents = publicKeyPem.replace("-----BEGIN PUBLIC KEY-----", "").replace("-----END PUBLIC KEY-----", "").trim();
-  const binaryDer = Uint8Array.from(atob(pemContents), (c) => c.charCodeAt(0));
+  try {
+    const pemContents = publicKeyPem.replace("-----BEGIN PUBLIC KEY-----", "").replace("-----END PUBLIC KEY-----", "").trim();
+    const binaryDer = Uint8Array.from(atob(pemContents), (c) => c.charCodeAt(0));
 
-  const publicKey = await crypto.subtle.importKey(
-    "spki",
-    binaryDer,
-    {
-      name: "RSASSA-PKCS1-v1_5",
-      hash: "SHA-256",
-    },
-    true,
-    ["verify"]
-  );
+    const publicKey = await crypto.subtle.importKey(
+      "spki",
+      binaryDer,
+      {
+        name: "RSASSA-PKCS1-v1_5",
+        hash: "SHA-256",
+      },
+      true,
+      ["verify"]
+    );
 
-  const signatureArray = Uint8Array.from(atob(signature), (c) => c.charCodeAt(0));
-  const dataArray = new TextEncoder().encode(JSON.stringify(payload));
+    const signatureArray = Uint8Array.from(atob(signature), (c) => c.charCodeAt(0));
+    const dataArray = new TextEncoder().encode(JSON.stringify(payload));
```

Comment:
Run formatting

Reply:
what's wrong with the formatting? I ran `format:prettier` and the formatting doesn't change